### ISSUE TITLE
Class PduUtility not found

### ISSUE THREAD

**Issue by vb559** [SmsNica/pdu#2]
Hi, Running the code throws the error 'Class PduUtility not found' (self::$instance = new PduUtility(); on line 123 in Pdu/Pdu.php) ?

**Comment by BruceLampson** [comment]
Hi, are you using any type of autoloading strategy?

**Comment by vb559** [comment]
No, but 'PduUtility' class is not defined in any of the files provided in src/ directory.
